webassembly/jszip.min.js

```diff
@@ -0,0 +1,13 @@
+/*!
+
+JSZip v3.3.0 - A JavaScript class for generating and reading zip files
+<http://stuartk.com/jszip>
+
+(c) 2009-2016 Stuart Knightley <stuart [at] stuartk.com>
+Dual licenced under the MIT license or GPLv3. See https://raw.github.com/Stuk/jszip/master/LICENSE.markdown.
+
```

Comment:
I assume you can just apply either license and MIT license should be ok for this project?